endpoints.php

```diff
@@ -308,6 +317,13 @@
             'permission_callback' => 'TapestryPermissions::putTapestrySettings',
         ],
     ],
+    'KALTURA_TEST' => (object) [
```

Comment:
Please do not leave TEST code in here. If this is not for testing, please change its name to something more meaningful like `TRANSFER_VIDEOS_TO_KALTURA` or something. 

For security purposes, it might also be a good idea to add a hashed url parameter that acts as a secret for running this operation. You can check that using a custom `permission_callback`.